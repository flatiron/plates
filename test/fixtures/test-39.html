<div class="row-fluid content">
  
  <div class="span4" role="complementary">
    <h2>Authors</h2>
    <div class="author"></div>
  </div>
    
</div>