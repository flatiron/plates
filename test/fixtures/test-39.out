<div class="row-fluid content">
  
  <div class="span4" role="complementary">
    <h2>Authors</h2>
    <div class="author">ABC</div><div class="author">DEF</div>
  </div>
    
</div>